README.rst

```diff
@@ -53,35 +53,13 @@ Features
 Installation
 ============
 
-Strawberry Fields requires Python version 3.5, 3.6, or 3.7 (3.8 is currently not support). Installation of Strawberry Fields, as well as all dependencies, can be done using pip:
+Strawberry Fields requires Python version 3.5, 3.6, 3.7, or 3.8. Installation of Strawberry Fields, as well as all dependencies, can be done using pip:
 
 .. code-block:: bash
 
     pip install strawberryfields
 
 
-TensorFlow support
```

Comment:
Will support for TF 1 be discontinued completely?

Reply:
Yep! The TF1.3 backend was too outdated (current release is 1.17 I think), and no longer works well with Python 3.7+. To keep TF 1.X support, we would need to upgrade the TF1.3 backend to TF1.17, which is not worth it.